sonar-db/src/main/java/org/sonar/db/DdlUtils.java

```diff
@@ -42,7 +42,10 @@ public static boolean supportsDialect(String dialect) {
   /**
    * The connection is commited in this method but not closed.
    */
-  public static void createSchema(Connection connection, String dialect) {
+  public static void createSchema(Connection connection, String dialect, boolean createSchemaMigrations) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Provide multiple methods instead of using "createSchemaMigrations" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2301)